CHANGELOG.md

```diff
@@ -1,5 +1,70 @@
 # Changelog
 
+## v0.1.55
+
+<!-- generated comparing v0.1.54..master -->
+
+_May 8, 2024_
+
+A big thanks to the 7 contributors who made this release possible. Here are some highlights ✨:
+
+Add max width option to page container, auto-scroll to selected elements, support enabling/disabling initial visibility in Data Grid columns, environment variable preview fixes, fix functions on Windows, add deployment guides for some other platforms.
+
+- &#8203;<!-- 52 -->Add max width option to the page (#3484) @Janpot
+- &#8203;<!-- 51 -->Changes required for the React integration (#3493) @Janpot
+- &#8203;<!-- 50 -->Update renovate.json @Janpot
```

Comment:
Let's remove these?

Reply:
You mean just these? https://github.com/mui/mui-toolpad/pull/3532/commits/380fe5d4da209f65a90511c77390198f542e248b